resources/assets/less/components/gistlog.less

```diff
@@ -144,6 +144,15 @@
         top: 0.5em;
         width: 2em;
     }
+
+    table td {
+        padding: 6px 13px;
```

Comment:
1. what do you think about ems instead of pixels?
2. alpha sort on css, so border first and then padding
3. is this a color we've used before? if so--or even if not--let's make it a variable so we can keep our number of colors down
4. could you do this nested? table { tr {} td {} }

Great stuff dude!

Reply:
No problem, should have it in a few